src_c/key.c

```diff
@@ -102,10 +105,20 @@ pg_scancodewrapper_repr(pgScancodeWrapper *self)
     return ret;
 }
 
+static PyObject *
+pg_iter_raise(PyObject *self)
+{
+    PyErr_SetString(pgExc_IterNotSupportedError,
+                    "Iterating over key states is no long supported");
+    return NULL;
+}
+
 static PyTypeObject pgScancodeWrapper_Type = {
     PyVarObject_HEAD_INIT(NULL, 0).tp_name = _PG_SCANCODEWRAPPER_TYPE_FULLNAME,
     .tp_repr = (reprfunc)pg_scancodewrapper_repr,
     .tp_as_mapping = &pg_scancodewrapper_mapping,
+    .tp_iter = &pg_iter_raise,
+    .tp_iternext = &pg_iter_raise,
```

Comment:
The `&` in both these lines is not needed: `tp_iter` and `tp_iternext` need function pointers, and not function pointer to pointers